tests/integration/components/welcome-page-test.js

```diff
@@ -1,60 +1,58 @@
-import { VERSION } from '@ember/version';
-import { moduleForComponent, test } from 'ember-qunit';
+/* eslint-disable ember/new-module-imports */
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
```

Comment:
Can we update the import statement to a named import?
```suggestion
import { hbs } from 'ember-cli-htmlbars';
```

Reply:
Seems fine to me 👍